src/galaxy/app.py

```diff
@@ -177,7 +178,8 @@ class Insight:
     """This class connects to Insight database and responsible for all the Insight related functionality"""
 
     def __init__(self, parameters=None):
-        self.database = Database(dict(config.items("INSIGHTS_PG")))
+        self.database = get_db_connection_params("INSIGHTS")
```

Comment:
This function should be passed to Database class so that database can connect with credentials provided from get_db_connection_params

Reply:
Also we need to import this function otherwise it will not be executable